frontend/src/components/app-sidebar.tsx

```diff
@@ -34,54 +34,62 @@ interface NavigationSection {
   items: NavigationItem[];
 }
 
-const navigationItems: NavigationSection[] = [
-  {
-    title: "Dashboard",
-    items: [
-      {
-        title: "Overview",
-        url: "/overview",
-        icon: Eye,
-      },
-      {
-        title: "Channels",
-        url: "/channels",
-        icon: Socket,
-        count: 10,
-      },
-      {
-        title: "All Nodes",
-        url: "/nodes",
-        icon: Graph,
-        count: 10,
-      },
-      {
-        title: "Events",
-        url: "/events",
-        icon: Network,
-        count: 10,
-      },
-    ],
-  },
-  {
-    title: "Transactions",
-    items: [
-      {
-        title: "Payment",
-        url: "/payments",
-        icon: Note,
-      },
-      {
-        title: "Invoices",
-        url: "/invoices",
-        icon: Binoculars,
-      },
-    ],
-  },
-];
-
 export function AppSidebar({ ...props }: React.ComponentProps<typeof Sidebar>) {
   const pathname = usePathname();
+  const [channelCount, setChannelCount] = useState(0);
+
+  useEffect(() => {
+    const fetchChannelCount = async () => {
+      try {
+        const res = await fetch("/api/channels?page=1&per_page=10");
```

Comment:
we should not hardcode filter query params. the range should be specified by the user from the UI.
except where there is pagination and user can move the next page.